### PR TITLE
first contribution

### PR DESCRIPTION
**Pull request by fulges**
I added one contribution

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,7 @@
+*
```
Comment (aleianno90) [inline .gitignore:1]: Why this gitignore?

File: README.md
Comment (aleianno90) [inline README.md:1]: This file is empty.

**Review by aleianno90 — CHANGES_REQUESTED** [review]
minor comments

**Comment by fulges** [comment]
I added a second contribution to answer your comments

**Review by aleianno90 — APPROVED** [review]

**Review by aleianno90 — APPROVED** [review]

**Pull request merged**